doc/how_to/load_matalb_data.rst

```diff
@@ -0,0 +1,66 @@
+Exporting MATLAB Data to Binary & Loading in SpikeInterface
+===========================================================
+
+In this tutorial, we'll go through the process of exporting your data from MATLAB in a binary format and then loading it using SpikeInterface in Python. Let's break down the steps.
+
+Exporting Data from MATLAB
+--------------------------
+
+First, ensure your data is structured correctly. The data matrix should be organized such that the first dimension corresponds to samples/time and the second dimension to channels.
+
+.. code-block:: matlab
+
+   % Define the size of your data
+   num_samples = 1000;
+   num_channels = 384;
+
+   % Generate random data as an example
+   data = rand(num_samples, num_channels);
+
+   % Write the data to a binary file
+   fileID = fopen('your_data_as_a_binary.bin', 'wb');
+   fwrite(fileID, data, 'double');
+   fclose(fileID);
+
+.. note::
+
+   In a real-world scenario, replace the random data generation with your actual data.
+
+Loading Data in SpikeInterface
+-----------------------------
+
+This should produce a binary file called `your_data_as_a_binary.bin` in your current MATLAB directory.
+You will need the complete path (i.e. its location on your computer) to load it in Python.
+
+Once you have your data in a binary format, you can seamlessly load it into SpikeInterface using the following script:
+
+.. code-block:: python
+
+   from spikeinterface.core.binaryrecordingextractor import BinaryRecordingExtractor
+   from pathlib import Path
+
+   # Define the path to your binary file
+   file_path = Path("/The/Path/To/Your/Data/your_data_as_a_binary.bin")
+
+   # Ensure the file exists
+   assert file_path.is_file()
+
+   # Specify the parameters of your recording
+   sampling_frequency = 30_000.0  # in Hz, adjust as per your matlab dataset
+   num_channels = 384  # adjust as per your matlab dataset
+   dtype = "float64"
+
+   # Load the data using SpikeInterface
+   recording = BinaryRecordingExtractor(file_path, sampling_frequency=sampling_frequency,
+                                        num_channels=num_channels, dtype=dtype, gain_to_uV=1, offset_to_uV=0)
```

Comment:
I wonder if we need to warn them about `gain_to_uV` and `offset_to_uV`. Maybe just a comment saying that these are used to convert to the actual voltages in case their data came from a reader that returned "proprietary units" that haven't been converted yet?

Reply:
Well the use case could be that the user have their data in `int16`. In that case, gains and offsets are needed to correctly convert to uV. I think it's an important concept to spend a couple of words on and it is related to the task!

Reply:
I like the idea of having a super goal oriented tutorial to get the job done. But in this case those two values are (at least in my opinion) important for the task at hand. As @alejoe91 said in the case the data was stored as an `int16` and needs to converted into a voltage. I think at a minimum a comment saying that not all data are in the correct format for all sorters so `gain_to_uV` and `offset_to_uV` will give you this fine control and then a link to relevant documentation that explains how these options work. Although in this case I tend toward educating (or maybe you'd argue over-educating) the user rather than overwhelming the user.

Reply:
`gain_to_uv` and `offset_to_uv` are an implementation detail of data extraction. My fear is that most users will not be familiar with them and that they already will have the data in the right units (as they are using that in MATLAB probably to analyze). Introducing these concepts to new users -as I was one- is more likely to raise more questions, confuse and derail. I don't think we can get away with just a few comments.

I think we can meet in the middle. I added a specific section at the end dealing with integer typed traces, this separate the information streams and does not get in the way of new users and gives us more space to introduce the necessary context for using gains and offsets. Could you check it @alejoe91 and @zm711 ?

Reply:
I'm fine with that section. It fits with the design I like (hive off the extra info for those that need it, but give an off-ramp for those who don't). Thanks @h-mayorquin :)

Reply:
Great. Do check the video linked above if you haven't before. I think is very good by the way.

Reply:
It is now bookmarked. I'll check it out this afternoon :)